knowledge_repo/app/templates/post_editor_base.html

```diff
@@ -8,12 +8,10 @@
 {% block title %} Knowledge {% endblock %}
 
 {% block content %}
-    {{ super() }}
-
-    <div class="panel-webpost container">
-      <div class="col-md-12"  id="#header-panel">
-        <h3> {{ path if path else 'New Knowledge Post' }}
-
+  {{ super() }}
+  <div class="col-lg-14">
```

Comment:
Are there non formatting changes in this file other than this line? It's tough for me to tell

Reply:
Yes, besides this line, it's mostly formatting changes:
- Indenting the code for readability
- Added a missing div tag

Changing this line from 12 to 14 fixed the alignment of the change. When set to 12, the form was aligned to the right of the page.